src/App.tsx

```diff
@@ -1,16 +1,23 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [initialMovies, setInitialMovies] = useState(moviesFromServer);
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={initialMovies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie
+          onAdd={(movie) => {
+            setInitialMovies((prevMovies) => [...prevMovies, movie]);
+          }}
+        />
```

Comment:
It's bad practice use anonymous function in JSX. Create separate handler and pass reference here
```
const onMovieAdd = (movie) => {
  setInitialMovies((prevMovies) => [...prevMovies, movie]);
}
```

```suggestion
        <NewMovie onAdd={onMovieAdd} />
```